views/displaycard/style.css

```diff
@@ -0,0 +1,24 @@
+#profile-owner-name {
+    color: #ffc107;
+}
+
+.profile-info-div {
+    background-color: white;
+    border-radius: 10px;
```

Comment:
It's good practice to use pixel values for things that probably won't change too much like border-radius and image sizes, but I would use rem or something similar for values like `margin-top` that deal with actually positioning.

I'd have to look into it but I wonder if you can apply a bootstrap class to an element in the js file. If so, it would limit how much of the css you'd have to write yourself here.

Reply:
>do you recommend a class that you'd want to apply to the element to mirror the same effect?

The glory of bootstrap is that it saves you time because instead of having to create a css class to make the background white, you can instead use a class called "bg-white" to do so. There are times in which you will need to use a custom class, but I just wanted to make sure you knew that bootstrap can do some of the things that you're putting into the class, such as applying margin spacing or padding.

>  I looked through and the closest one I found was "card"

Yea if you wanted to bootstrapify the forms you could use a card but if not, it's also fine. For stuff like that, you'd manually add spacing to each div by messing with the margin and padded settings. 

Like I said though, it's not a big deal if you don't change it so do whatever makes your life easier